services/ingestion/etl/2014tc16rfir001/xls/src/lib/transform.js

```diff
@@ -0,0 +1,311 @@
+// @flow
+
+import sanitizeBudgetItem from '@eubfr/lib/budget/budgetFormatter';
+import type { Project } from '@eubfr/types';
+
+/**
+ * Preprocess `budget`.
+ *
+ * Input fields taken from the `record` are:
+ *
+ * - `Total eligible expenditure allocated to the operation / Montant total éligible attribué au projet`
+ * - `Union co-financing rate as per priority axes / Taux de cofinancement de l'UE selon le statut du bénéficiaire`
+ *
+ * @memberof 2014tc16rfir001XlsTransform
+ * @param {Object} record The row received from parsed file
+ * @returns {Budget}
+ */
+
+const getBudget = record => {
+  let total = 0;
+  let euContrib = 0;
+
+  const totalCosts = record[
+    'Total eligible expenditure allocated to the operation / Montant total éligible attribué au projet'
+  ].split(';');
+
+  totalCosts.forEach(cost => (total += Number(cost))); // eslint-disable-line
```

Comment:
still don't want to use `.reduce()`? 👼

Reply:
I think it's more readable like this, no?